.gitignore

```diff
@@ -1,11 +1,2 @@
-# Dependency directory
-# Commenting this out is preferred by some people, see
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
-client/node_modules
-client/bower_components
-
-
-# Webstorm folder
-.idea
-
-client/**/*.css
\ No newline at end of file
+#Webstorm IDE VERY VERY IMPORTANT
+.idea
```

Comment:
How about you knock off the passive-aggressive bs and we can move on with our lives?